src/radio-button.jsx

```diff
@@ -11,8 +11,9 @@ const RadioButton = React.createClass({
 
   propTypes: {
     /**
-     * Checked if true.
+     * Used internally by `RadioButtonGroup`.
      */
+    /* Checked if true. */
```

Comment:
It was a workaround until we have @internal support. "Checked if true" isn't of interest in the public docs.